srcpkgs/nicotine+/template

```diff
@@ -1,15 +1,15 @@
 # Template file for 'nicotine+'
 pkgname=nicotine+
-version=3.2.9
-revision=2
+version=3.3.4
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools gettext"
 depends="gspell gtk+3 python3-gobject"
 checkdepends="$depends python3-pytest python3-pytest-xvfb"
 short_desc="Graphical client for the Soulseek peer-to-peer network"
-maintainer="doggone <doggone@airmail.cc>"
+maintainer="doggone <doggone@airmail.cc>, Fabian Kaiser <fk_development@protonmail.com"
```

Comment:
we'd prefer packages be adopted by established contributors. I encourage you to keep track of this package and continue making contributions, but please don't change the maintainer at this time.

Additionally, you do not need to be the maintainer of a package to update it, and "co-maintainership" is not typically done on void

https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#adopting-a-template